modules/ROOT/pages/publishing-a-rest-api.adoc

```diff
@@ -0,0 +1,16 @@
+= Publishing a REST API
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+Mule provides a dedicated toolkit to facilitate the development and exposure of REST APIs. APIkit Rest is a simple framework that helps you with API-first development, enforcing good API implementation practices. You can add this toolkit to your instance of Anypoint Studio. +
```

Comment:
"Mule provides"  >  "Mule runtime engine provides you with...."  [product naming; direct address, if appropriate]

Is a "dedicated toolkit" different from any other type of toolkit? What are we trying to emphasize here?

"...to facilitate the development and exposure of REST APIs."
   >  "...to help you develop and expose REST APIs."   [active voice, if appropriate; simpler wording]

I did some searching around and could find nothing that refers to "APIkit Rest." 
I did, however, discover this topic: https://docs.mulesoft.com/apikit/4.x/.
Given this existing concept topic about APIkit, perhaps this topic can make some assumptions about audience knowledge and jump right in with something like this:

"After you add APIkit to your Anypoint Studio instance, you can rely on it to help you easily develop and expose REST APIs, following best API implementation practices."
or
"Adding APIkit to your Anypoint Studio instance enables you to quickly and properly develop, document, and test your Mule APIs."

Big Question: Do we really need this topic, given https://docs.mulesoft.com/apikit/4.x/?